app/javascript/App/components/NewsfeedPage/NewsListHeader.js

```diff
@@ -15,6 +15,7 @@ const NewsListHeader = (props) => {
     coins,
     feedSources,
     activeFilters,
+    activeEntity,
```

Comment:
Is `activeEntity` used in the component?

Reply:
good question, from here moving forward the linter will prevent unused vars